### PR TITLE
[ENHANCEMENT] dropRight bad example 👎

### PR DESCRIPTION
**Pull request by retyui**
## Description
The slice function can take the negative negative value of the second agrimement

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by retyui** [comment]
@Chalarangelo 
What are the exceptions?
You are sure ?

image

**Comment by Chalarangelo** [comment]
I ma not sure because I am on mobile so I cannot test this right now, however it *seemed* like it would not handle the edge cases it used to, but your screenshot proves it does. Can someone else review and merge this PR if it works?

**Comment by skatcat31** [comment]
This should pass all test cases, but it needs the documentation updated. slice by design of the negative index implementation is dropRight( [see MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) )

File: snippets/dropRight.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Returns a new array with `n` elements removed from the right
 Check if `n` is shorter than the given array and use `Array.slice()` to slice it accordingly or return an empty array.
```
Comment (skatcat31) [inline snippets/dropRight.md]: Please update the description

**Comment by Chalarangelo** [comment]
I agree with @skatcat31, after updating the description we should be ready to merge this!

**Comment by retyui** [comment]
@Chalarangelo My english skill very bad. You can update the documentation yourself ?

**Comment by Chalarangelo** [comment]
@retyui No problem at all! 👍
